src/marker_traits.rs

```diff
@@ -31,7 +31,7 @@ pub trait Ord {
 /// The **marker trait** for compile time bits.
 ///
 /// This trait should not be implemented for anything outside this crate.
-pub trait Bit {
+pub trait Bit: Copy + Default {
```

Comment:
Since we'll be working extensively with `Bit`s in the value domain, it's useful for `Bit` to imply `Copy` (so we don't have to explicitly clone the ZSTs, or swamp typenum with trait bounds) and `Default` (so we can safely instantiate new values).

Likewise with `Unsigned`.